src/utils/compute-util.ts

```diff
@@ -1,5 +1,5 @@
 // to-do: compute-go doesn't seem to replace old bundles. Generate a unique
 //        hostname for each new build.
 export function newHostname(name: string, version: number): string {
-  return `${name.toLocaleLowerCase().replace(/[^a-z0-9]/, '') || 'untitled'}-${version}.local`
+  return `${name.toLocaleLowerCase().replace(/[^a-z0-9]/, '') || 'untitled'}-${version}-local`
```

Comment:
I was mistakenly under the impression that this hostname had to end in .local for compute. Can you just delete the entire `-local`? I don't think it's adding any value.

Reply:
sounds good! deleted, re-confirmed it still works as expected